paymentsheet/src/main/java/com/stripe/android/paymentsheet/repositories/CustomerAdapter.kt

```diff
@@ -0,0 +1,72 @@
+package com.stripe.android.paymentsheet.repositories
+
+import androidx.annotation.RestrictTo
+import com.stripe.android.ExperimentalSavedPaymentMethodsApi
+import com.stripe.android.model.PaymentMethod
+
+@ExperimentalSavedPaymentMethodsApi
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+interface CustomerAdapter {
```

Comment:
Why is this an `ExperimentalSavedPaymentMethodsApi`? Are we thinking of making this public?

Reply:
Yes, `CustomerAdapter` will be part of the public API which allows integrators to build their own solution for saving/retrieving customer payment methods. We will offer a default implementation that they will need to construct.

Reply:
This is the name that iOS is going with, I would like to keep it mostly consistent. But if you feel very strongly about it, we can change it. Also, this isn't set it stone, we can bring attention to the naming in the API review.